helm/reana/values.yaml

```diff
@@ -113,3 +113,7 @@ traefik:
   ssl:
     enabled: true
     generateTLS: true
+
+# Quota
+quota:
+  disk_update: "0 3 * * *" # everyday at 3am
```

Comment:
TBD. I chose a _random_ night time, perhaps it's better to run it pretty early in the morning so users can have fresh quotas? 6am? 7am?

Reply:
The users can be living in all time zones (e.g. West Coast USA), so calculating around 2am-5am is usually the best.  One thing to pay attention to is to "calculate until midnight past day" so that users have some clear sharp time reference.  BTW we could also display this information visibly in the dashboard, for example "Last updated 2020-10-14-23:59:59 GMT" as a grey note near the quota charts somewhere on that page.

Reply:
Shall we create an issue to implement this?